.gitignore

```diff
@@ -10,6 +10,7 @@ test-results/
 playwright-report/
 Sources/ContentScopeScripts/dist/
 test-results
+!Sources/ContentScopeScripts/dist/pages/.gitignore
```

Comment:
the swift module is technically invalid whilst the `pages` folder is absent. this fixes it by ensuring the empty folder is there.